src/instructlab/configuration.py

```diff
@@ -457,9 +457,9 @@ class _train(BaseModel):
     # TODO: could move into its own object.
     # Not strictly necessary for a correct training object.
     phased_phase1_num_epochs: int | None = Field(
-        default=10,
+        default=7,
```

Comment:
nit: Might be good to move 7 to a constant given the number of references

Reply:
I'll make an issue for that, you're right